docs/source/asgi/examples/starlette/app.py

```diff
@@ -7,7 +7,13 @@
 
 # The `allowed_hosts` argument is required when running under HTTPS. It's
 # used for additional CSRF defence.
-admin = create_admin([Director, Movie], allowed_hosts=["my_site.com"])
+admin = create_admin(
+    [Director, Movie],
+    allowed_hosts=["my_site.com"],
+    # Required for change password
+    # and create new user endpoints:
+    prefix_path="/admin/",
```

Comment:
I see the issue here. We need to redirect the user to the login page, but we don't know what URL the admin is mounted at. It means we have to do soething like:

```python
app = FastAPI()

app.mount('/admin/', create_admin([Director, Movie], prefix_path="/admin/")
``` 

Repeating the path is a bit unfortunate. I wonder if there's another way around it?

As we're calling the endpoints via REST, can we just ignore the redirect coming from the server, and just redirect from the front end instead?

Reply:
Yes. This is a problem and I don't know how to properly solve it. If you have any ideas feel free to apply them.

Reply:
Somehow I managed to solve it with this:
```python
route=change_password(login_url="./../../auth/login/"),
```
It's not the nicest, but it works and we don't have to use `prefix_path` anymore in the `create_admin` function.

Reply:
That's better. Does the `login_url` matter at all though? We're not using that HTML form - just via the API.

Reply:
This is important because if we leave the default `login_url` of the password change endpoint, we will be redirected to `https://some-url/login`, but we change the admin user password and only the admin (`admin_only=True` in `auth_middleware`) can do that. For example
```python
app = FastAPI()

app.mount('/admin/', create_admin([Director, Movie]) 
```
need `https://some-url/admin/auth/login` not `https://some-url/login` to properly work.
If we change mount path to e.g. `piccolo-admin` we need `https://some-url/piccolo-admin/auth/login` not `https://some-url/login`. I hope that makes sense.